UPGRADING.md

```diff
@@ -0,0 +1,179 @@
+
+**The new 'webpack.config.js file'**
+
+In `webpacker` this file was hidden inside the gem, but now you can edit it directly. If you had custom configuration in `config/webpack` you can move them to here. Projects with multiple entrypoints will need to adjust the `entry` attribute:
+
+```js
+module.exports = {
+  entry: {
+    application: "./app/javascript/application.js",
+    admin: "./app/javascript/admin.js"
+  }
+}
+```
+
+**The 'link_tree' directive added to 'app/assets/manifest.js'**
+
+This tells Sprockets to include the files in `app/assets/builds` during `assets:precompile`. This is the folder where `yarn build` will place the bundled files, so make sure you commit it to the repository and don't delete it when cleaning assets. 
+
+**What about babel?**
+
+If you would like to continue using babel for transpiling, you will need to configure it manually. First, open `webpack.config.js` and add this:
+
+```js
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.(js)$/,
+        exclude: /node_modules/,
+        use: ['babel-loader']
+      }
+    ]
+  }
+}
+```
+
+Then open `packages.json` and add this:
+```json
+"babel": {
+  "presets": [
+    "./webpack.babel.js"
+  ]
+}
+```
+
+Finally, download [webpackers babel preset](https://github.com/rails/webpacker/blob/master/package/babel/preset.js) file and place it in the same directory as `packages.json` with the name `webpack.babel.js`.
+
+## 2. Migrate from sass-rails to cssbundling-rails
+
+Start by following these steps:
+
+1. Add `cssbundling-rails` to your Gemfile (do not remove `sass-rails`, Sprockets still needs it);
+2. Run `./bin/bundle install`;
+3. Run `./bin/rails css:install:sass`.
+
+After you are done you will notice that the install step updated some files.
+
+**The new process in 'Procfile.dev'**
+
+Just like the javascript process, this one will bundle and watch for changes in css files.
+
+**The 'build:css' attribute added to packages.json**
+
+This is the command `yarn build` will use to bundle css files.
+
+**The 'link_tree' directive removed from 'app/assets/manifest.js'**
+
+Now that the CSS files will be placed into `app/assets/build`, Sprockets no longer needs to worry about the `app/assets/stylesheets` folder. If you have any other `link_tree` for css files, remove them too.
+
+**Configuring multiple entrypoints**
+
+Sprockets will only compile files in the root directories listed in `manifest.js`, but the sass package that `yarn build` uses will also check subfolders, which might cause compilation errors if your scss files are using features like `@import` and variables. This means that if you have multiple entry points in your app, you have some extra work ahead of you. 
+
+Let's assume you have the following structure in your `app/asset/stylesheets` folder:
+
+```
+stylesheets/admin.scss
+stylesheets/admin/source_1.scss
+stylesheets/admin/source_2.scss
+stylesheets/application.csss
```

Comment:
@brenogazzola Little typo on the extension here 😊

Reply:
Fixed, thanks